assets/tests/components/mapPage.test.tsx

```diff
@@ -329,8 +360,11 @@ describe("<MapPage />", () => {
     await userEvent.click(screen.getByRole("button", { name: /close/i }))
     expect(vehiclePropertiesCard).not.toBeInTheDocument()
     expect(routeShape).not.toBeInTheDocument()
-
-    expect(mapSearchPanel).toBeVisible()
+    expect(
+      screen.getByRole("generic", {
+        name: /map search panel/i,
+      })
+    ).toBeVisible()
```

Comment:
question: This is having to be reaquired because of rerendering, right? do you know why it's being invalidated/recreated?

I'm noticing on dev blue that the map page blinks and resets a bunch of state when I select a new vehicle. If possible, I think it'd be best if we prevented it from destroying and recreating these objects when rendering?